src/main/app/src/app/informatie-objecten/model/file-icon.ts

```diff
@@ -49,4 +49,18 @@ export class FileIcon {
     compare(other: FileIcon) {
         return this.type.localeCompare(other.type);
     }
+
+    static getIconByBestandsnaam(bestandsnaam) {
```

Comment:
Dus blijkbaar hadden we al een icoontje voor elk bestandstype? Da's mooi, maar was er niet iets met dat we dat op een bepaalde plaats(en) nog niet gebruiken? (Als ik me goed herinner.) Dat zie ik niet in je aanpassingen...

Reply:
Ik zie de tekst `fa-file` in de front-end verder alleen voorkomen in `abstract-file-form-field-builder.ts`, maar dat is in een andere context. Ik ga morgen bij de standup even algemeen vragen of iemand zo 123 weet of dit nog ergens anders wordt gebruikt wat ik over het hoofd zie.